docs/src/css/custom.css

```
@@ -1,103 +1,196 @@
   --primary-neutral-600: #c4c4c4;
+  --primary-neutral-800: #9da4ae;
+  --primary-blue-600: var(--secondary-blue-900);
+  --secondary-blue-900: #c6d6ff;
   --next-prev-border-color: #293441;
   --ifm-color-emphasis-100: #1d1e30;
   --ifm-color-emphasis-0: #111f3b;
 }
 
+/* stylelint-disable docusaurus/copyright-header */
+
+@import url("https://fonts.googleapis.com/css2?family=IBM+Plex+Mono&display=swap");
```

Biome (1.9.4)

[error] This @import is in the wrong position.

Any @import rules must precede all other valid at-rules and style rules in a stylesheet (ignoring @charset and @layer), or else the @import rule is invalid.
Consider moving import position.

(lint/correctness/noInvalidPositionAtImportRule)